config.go

```diff
@@ -92,7 +92,7 @@ type Settings struct {
 	Notify              NotifyConfig            `yaml:"notify" bson:"notify" json:"notify" id:"notify"`
 	Plugins             PluginConfig            `yaml:"plugins" bson:"plugins" json:"plugins"`
 	PluginsNew          util.KeyValuePairSlice  `yaml:"plugins_new" bson:"plugins_new" json:"plugins_new"`
-	PodInit             PodInitConfig           `yaml:"pod_init" bson:"pod_init" json:"pod_init" id:"pod_init"`
+	PodLifecycle        PodLifecycleConfig      `yaml:"pod_lifecycle" bson:"pod_lifecycle" json:"pod_lifecycle" id:"pod_lifecycle"`
```

Comment:
Nit: If there are entries in the db with the old bson tag, we may want to delete them to avoid hitting edge cases, especially on the UI.

Reply:
Yeah, I plan on deleting the old document once this is merged.